index.js

```diff
@@ -472,8 +472,8 @@ app.post('/event/join', (req,res) => {
               'JOIN',
               result.rows[0],                                                       //All event info
               {
-                login: req.user.data.login,                                         //Login of who joined
-                name: `${req.user.data.firstname} ${req.user.data.lastname}`        //Name of who joined
+                login: req.body.login,                                         //Login of who joined
+                name: `${req.body.firstname} ${req.body.lastname}`        //Name of who joined
```

Comment:
Socket.io uses these, they will not conflict with route testing code. We are not sending peoples names with every socket message